PHX-42 Css stylign changes in the edx-platform

muhhshoaib opened the pull request:
@chrisndodge PHX-42 Css changes.

chrisndodge commented:
Hi @muhhshoaib I'm seeing a visual regression regarding the background of the sequence on all sequences:

![screen shot 2015-06-30 at 11 03 11 am](https://cloud.githubusercontent.com/assets/2362917/8434154/bfc3b2ca-1f17-11e5-9fee-be7bbd003f63.png)

muhhshoaib commented:
@chrisndodge yes, there were styling class conflicts. I have added the new class names.
I have also added the css for the PHX-43 (Implement the proctored start page) in this PR.
